FakeServer/Common/Formatters/XmlOutputFormatter.cs

```diff
@@ -24,20 +25,19 @@ public XmlOutputFormatter()
         protected override bool CanWriteType(Type type) =>
                 typeof(ExpandoObject).IsAssignableFrom(type) || typeof(IEnumerable<object>).IsAssignableFrom(type) ? base.CanWriteType(type) : false;
 
+        private static Pluralizer pluralizer = new Pluralizer();
```

Comment:
This project uses underscore naming convention for private fields (or should use, there might be some places that don't follow this).

```cs
private static Pluralizer _pluralizer = new Pluralizer();
````